.github/workflows/release.yml

```diff
@@ -213,6 +219,8 @@ jobs:
             --alias linux-arm=aarch64-unknown-linux-gnu:tar.gz \
             --alias windows=x86_64-pc-windows-msvc:zip \
             --alias windows-tar=x86_64-pc-windows-msvc:tar.gz \
+            --alias windows-32=i686-pc-windows-msvc:zip \
+            --alias windows-32-tar=i686-pc-windows-msvc:tar.gz \
```

Comment:
Thanks! Can we rename these to `windows-x86` and `windows-x86-tar`?

Reply:
Done